pipelines/core-services/csi-secret.yaml

```diff
@@ -0,0 +1,5 @@
+- op: add
+  path: /spec/tasks/2/params
+  value:
+    - name: shared-secret
+      value: redhat-appstudio-staginguser
```

Comment:
this is for wrong task should be 0 now, also this is not needed at all. We will rather link secret to serviceaccount instead of using shared secret. So please just remove this file (and link in kustomization.yaml)